posthog/hogql/ai.py

```diff
@@ -152,3 +142,20 @@ def write_sql_from_prompt(prompt: str, *, current_query: Optional[str] = None, t
         return candidate_sql
     else:
         raise PromptUnclear(error)
+
+
+def hit_openai(messages, user) -> tuple[str, int, int]:
+    result = openai.chat.completions.create(
+        model="gpt-4o",
```

Comment:
Have you tried the gpt-4o-mini model? I'm wondering if it's sufficient for this task.

Reply:
That's a good idea. Just tried it out… and weirdly it seems quite a bit worse at coming with _concise_ questions. They seem to be a bit more verbose in general, and the reasoning beforehand is slightly less robust. Though this is challenging to actually gauge, since there's so much nondeterminism in LLMs. Will stick to 4o for top performance for now